pages/dkp/konvoy/2.2/Install/index.md

```diff
@@ -0,0 +1,102 @@
+Kubernetes.
+
+As an example, this Installation guide provides simplified instructions to get your DKP cluster up and running with minimal configuration requirements on an Amazon Web Services (AWS) public cloud instance. For information about installing on a different platform, see [Choose Infrastructure](../choose-infrastructure/). 
+
+# Before you begin
+
+Before installing DKP, ensure you have the [following](../supported-operating-systems/).
+
+# Installing DKP
+
+1.  Install required packages. In most cases, you can install the required software using your preferred package manager. For example, on a macOS computer, you can use [Homebrew][brew] to install `kubectl` and the `aws` command-line utility by running the following command:
+
+    ```bash
+    brew install kubernetes-cli awscli
+    ```
+
+1.  Check the Kubernetes client version. Many important Kubernetes functions **do not work** if your client is outdated. You can verify that the version of `kubectl` you have installed is supported by running the following command:
+
+    ```bash
+    kubectl version --short=true
+    ```
+
+1.  To download DKP, see the [Download Konvoy](../download) topic for information. You will need to download and extract the DKP binary package tarball.
+
+1.  Install with default settings.  The Container Network Interface (CNI) which is satisfied by Calico, the Container Storage Interface (CSI) which is satisfied by drivers specific to the cloud provider that's selected, CoreDNS (recursive DNS resolver that serves cluster Service Endpoints), and Cert Manager (for TLS Certificate management).
+
+1.  Verify you have valid **CLOUD PROVIDER security credentials** to deploy the cluster on that platform. This step is not required if you are installing DKP on an on-premises environment. For information about installing in an on-premises environment, see [Install on-premises](../choose-infrastructure/on-prem).
+
+1.  Create a directory for storing state information for your cluster by running the following commands:
+
+    ```bash
+    mkdir konvoy-quickstart
+    cd konvoy-quickstart
+    ```
+
+    This directory for state information is required for performing future operations on your cluster.
+    For example, state files stored in this directory are required to tear down a cluster.
+    If you were to delete the state information or this directory, destroying the cluster would require you to manually perform clean-up tasks.
+
+1.  Deploy with all of the default settings depending on which infrastructure you have.  Go to the Choose Infrastructure section of the documentation for further steps on creating a cluster on your Cloud platform. [Choose Infrastructure](../choose-infrastructure/)    
+
+# Next steps
+
+Now that you have a basic Konvoy cluster installed and ready to use, you might want to test operations by deploying
+a simple, sample application, customizing the cluster configuration, or checking the status of cluster components.
+
```

Comment:
There needs to be an additional before this one, which is is to install Kommander, as without it, all you have is a very basic Kubernetes cluster (the very bottom layer of the diagram).

Maybe we need to add an '# Install Kommander' bullet here with a pointer to the Kommander install directions?